appengine-vmruntime/setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='appengine-vm-runtime',
-    version='0.63',
+    version='0.64',
```

Comment:
Yes, but for now let's keep it in sync until we have a plan on how to increment this.  Can talk about this in our mtg.